ps_googleanalytics.php

```diff
@@ -139,11 +139,14 @@ public function hookDisplayAdminOrder()
     {
         $gaTagHandler = new PrestaShop\Module\Ps_Googleanalytics\Handler\GanalyticsJsHandler($this, $this->context);
 
-        echo $gaTagHandler->generate(
+        $output = '';
+        $output = $gaTagHandler->generate(
             $this->context->cookie->__get('ga_admin_refund'),
             true
         );
         $this->context->cookie->__unset('ga_admin_refund');
+
+        return $output;
```

Comment:
```suggestion
        return $gaTagHandler->generate(
            $this->context->cookie->__get('ga_admin_refund'),
            true
        );
        $this->context->cookie->__unset('ga_admin_refund');
```

But you should be sure it needs a return and not an echo :sweat_smile:

Reply:
Yes, echo does not work on migrated pages.
Maybe I am wrong, but with your suggestion, how will the cookie be unset, when you return before?
The code would then be run infinite times, hm?

Reply:
Ahah you're right, didn't see the cookie unset, badly scrolling :joy:

Reply:
About the echo, you should have a test on PS 1.6 at least to be sure :)